flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -333,42 +335,34 @@ static void _msp_send_altitude(struct msp_bridge *m)
 	msp_send(m, MSP_ALTITUDE, data.buf, sizeof(data));
 }
 
+static uint16_t _msp_scale_rc(float percent) {
```

Comment:
Could you please document this math equation? It will be hard to maintain without understanding its origins.